.travis.yml

```diff
@@ -99,47 +94,23 @@ jobs:
       name: 'End to End Tests'
       script:
         - unbuffer node build-system/pr-check/e2e-tests.js
-      env:
-        - CACHE_NAME=E2EJOB
     - stage: experiment
       name: 'Experiment A Tests'
       script:
         - unbuffer node build-system/pr-check/experiment-tests.js --experiment=experimentA
-      cache: false
     - stage: experiment
       name: 'Experiment B Tests'
       script:
         - unbuffer node build-system/pr-check/experiment-tests.js --experiment=experimentB
-      cache: false
     - stage: experiment
       name: 'Experiment C Tests'
       script:
         - unbuffer node build-system/pr-check/experiment-tests.js --experiment=experimentC
-      cache: false
     - stage: experiment
       name: 'Performance Tests'
       script:
         - unbuffer node build-system/pr-check/performance-tests.js
-      env:
-        - CACHE_NAME=PERFORMANCEJOB
   allow_failures:
     - script: unbuffer node build-system/pr-check/performance-tests.js # See #28148
     - script: unbuffer node build-system/pr-check/remote-tests.js # See #28343
   fast_finish: true
-before_cache:
-  # do not store cache for pr builds or experiment stage builds
-  - if [[ $TRAVIS_EVENT_TYPE == pull_request ]] || [[ $TRAVIS_BUILD_STAGE_NAME == experiment ]]; then exit $TRAVIS_TEST_RESULT ; fi
-cache:
-  directories:
-    - node_modules
-    - build-system/tasks/e2e/node_modules
-    - build-system/tasks/performance/node_modules
-    - build-system/tasks/visual-diff/node_modules
-    - sauce_connect
-    - validator/node_modules
-    - validator/nodejs/node_modules
-    - validator/webui/node_modules
-    - validator/java/bazel-installer
-    - $HOME/.m2
-    - .karma-cache
```

Comment:
This dir was being cached in order to speed up transforms during tests, which will now have to be done from scratch for every build.

/cc @jridgewell

Reply:
Maybe we could use the mechanism that shares stuff between build stages to build our own cache.

Reply:
Good idea. Definitely possible.